master/pkg/searcher/search_method.go

```diff
@@ -89,3 +91,8 @@ func (defaultSearchMethod) validationCompleted(
 func (defaultSearchMethod) trialClosed(context, RequestID) ([]Operation, error) {
 	return nil, nil
 }
+
+func (defaultSearchMethod) trialExitedEarly( //nolint: unused
```

Comment:
yes -- this is because this function in particular is not used (all our searchers "override" this function)

I kept this so that future search methods still have a "sane default" in case they opt to not implement the logic in to the search method